apps/daimo-mobile/src/view/shared/SearchHeader.tsx

```diff
@@ -41,23 +44,58 @@ export function SearchHeader({
 
   return (
     <View style={styles.header}>
-      <ButtonCircle size={50} margin={16} onPress={goToAccount}>
-        <AccountBubble eAcc={eAcc} size={50} fontSize={20} transparent />
-      </ButtonCircle>
+      <Animated.View
+        style={{
+          marginRight: 16,
+          height: 50,
+          justifyContent: "center",
+        }}
```

Comment:
curious, does the `StyleSheet.create` have any measureable impact vs plain objects?